src/ZEO/asyncio/base.py

```diff
@@ -129,13 +127,13 @@ def data_received(self, data):
                     self.getting_size = True
                     self.message_received(collected)
             except Exception:
+                #import traceback; traceback.print_exc()
```

Comment:
Yup. I'll remove.

(I wish Python has macros. I find myself adding and removing the same debugging code over and over. Sigh.)